ci-operator/config/openshift/machine-api-operator/openshift-machine-api-operator-master.yaml

```diff
@@ -118,6 +118,15 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-libvirt
+- as: e2e-vsphere
+  steps:
+    cluster_profile: vsphere
+    workflow: openshift-e2e-vsphere
+- as: e2e-vsphere-operator
+  commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e
+  openshift_installer_custom_test_image:
+    cluster_profile: vsphere
+    from: stable:cluster-api-actuator-pkg
```

Comment:
This isn't supposed to work already. It seems we are using multi-step vsphere job, and so you should run the tests similar to this example - https://github.com/openshift/release/blob/71ba7717f2ef5d0001bffe1185b800730230e9ac/ci-operator/config/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-master.yaml#L91-L103. You should also set the tests against `cluster-api-actuator-pkg` as I'm not sure if building a custom test image, pointing to other repos similar to what `openshift_installer_custom_test_image` provided is now supported. @jcpowermac Could you assist answering this question?